schemas/core/leg.json

```diff
@@ -2,27 +2,27 @@
   "$id": "http://maasglobal.com/core/leg.json",
   "type": "object",
   "description": "The base leg object with all fields, to be inherited",
-  "leg": {
```

Comment:
Fix should be to un-nest this "leg" property, so that the top-level definition is the content.